### DISCUSSION TITLE
MP4视频加载失败，flv不支持，大佬能否修复？

### DISCUSSION THREAD

**Discussion by fjlu** [libsgh/PanIndex#196] [Q&A]
大部分mp4加载失败，小部分可以放，不知道什么原因造成，flv能否更新支持，谢谢！！

**Comment by libsgh** [comment]
flv其实已经添加了，后面会优化下媒体类型的判断。

无论mp4/mkv/flv在web端播放都对音频视频编码有要求，请使用转码工具转成H264+AAC，再传到网盘播放
